### DISCUSSION TITLE
keystone.js 4 updateItem function throw error

### DISCUSSION THREAD

**Discussion by wendylw** [keystonejs/keystone#7789] [Questions] [answered]
error:
{ action: 'customer',
  loginessTimes: '1',
  email: 'a@a.com',
  _id: '62f595d9eda8d816dca82b96',
  type: 'login' }
TypeError: item.get is not a function
    at number.updateItem (/Users/xxx/dev/test/node_modules/keystone/fields/types/number/NumberType.js:127:25)
    at doFieldUpdate (/Users/xxx/dev/test/node_modules/keystone/lib/list/updateItem.js:149:20)

`Customer.updateItem(
				{
					_id: mongoose.Types.ObjectId(_id),
				},
				{
					_id: mongoose.Types.ObjectId(_id),
					loginessTimes: Number(loginessTimes),
				},
				{
					fields: "loginessTimes",
				},
				function (err) {
					if (err) {
						locals.validationErrors = err.errors;
					} else {
						locals.customerSubmitted = true;
					}
					next();
				}
			);`

**Comment by dcousens** [comment, marked as answer]
Hi @wendylw,

This is the `keystone-6` repository,  you can find keystone 4 at https://github.com/keystonejs/keystone-classic, but we (@keystonejs/maintainers) are not actively maintaining that project.
